package.json

```diff
@@ -159,6 +159,11 @@
         "title": "Push",
         "category": "OpenShift"
       },
+      {
```

Comment:
We can use the same command in palette and tree view, no new command declaration required.